packages/docs-markdown/src/controllers/yaml/showTOCQuickPick.ts

```diff
@@ -29,31 +39,33 @@ export async function showTOCQuickPick(options: boolean) {
 		return;
 	}
 
-	const fullPath = join(selectionPick.description, selectionPick.label);
+	if (selectionPick.description) {
+		fullPath = join(selectionPick.description, selectionPick.label);
+	}
+
 	const content = readFileSync(fullPath, 'utf8');
-	let title = getHeadings(content);
+	const headings = getHeadings(content);
 
-	if (!title) {
-		postError(`Could not find H1: ${fullPath}`);
-		showStatusMessage(`Could not find H1: ${fullPath}`);
+	if (!headings) {
+		window.showErrorMessage(headings[0]);
 		return;
 	}
-
+	let headingName = headings.toString().replace('# ', '').split(',')[0];
```

Comment:
```suggestion
	let headingName = headings.replace('# ', '').split(',')[0];
```

The `headings` variable should already be of type `string` according to [this](https://github.com/microsoft/vscode-docs-authoring/blob/74155c4528c8ce9ff19d8fa9a9dec7448c32be79/packages/docs-markdown/src/helper/getHeader.ts), no reason to call `.toString()`

Reply:
Hey @IEvangelist , thanks for the suggestions! I committed the previous suggestion but this one causes an issue because the headings are objects and not strings at this point. Here's the console output:

![image](https://user-images.githubusercontent.com/11825888/120550921-500fc700-c3aa-11eb-8980-75a766ab9f91.png)